src/app/spy-object/spy-object.spec.ts

```diff
@@ -9,9 +9,10 @@ describe('SpyObject', () => {
     person.sayHi.andReturn('Bye!');
   });
 
-  it('should keep getters/setters', () => {
+  it('should enable spying on properties', () => {
```

Comment:
Add this new ability to the `features` section in v4.

Reply:
👍